project-setup/communication-tooling.md

```diff
@@ -0,0 +1,84 @@
+# Title
+
+Communication tooling
+
+# Context
+
+A team depends on another team's component. It would like to make contributions
+to that component. Even when it happens in writing, communication happens in a
+1-on-1 fashion.
+
+# Problem
+
+A team is open to receiving contributions from downstream users of their
+component. Coordination and communication happens in an ad-hoc fashion though
+leading to incoherent information being shared, delays in answers received,
+contributors pinging multiple host team members before receiving a definitive
+answer.
+
+# Forces
+
+- The host team is interested in receiving contributions and willing to mentor
+  contributors.
+- Teams have a strong verbal communication culture and are inexperienced with
+  setting up project specific asynchronous communication channels.
+- Communication channels may be aligned with specific groups that should be
+  reached but not by communication purpose.
+
+# Solution
+
+The host team needs to be clear on the benefit of providing company-public,
+archived, searchable, linkable communication channels that are free to subscribe
+to by anyone in the company:
+
+- Others lurking and reading lowers the barrier to get involved raising the
+  likelihood of receiving contributions.
+- Answering questions in those channels means that not only other team members
+  can listen in and provide additional information, it also means that other
+  uses with the same question see (or later on find) the previous answer leading
+  to a lower need to repeat explanations.
+- Keeping communication in asynchronous channels allows for participants on
+  different schedules - either due to different time zones or due to different
+  routines - to meaningfully contribute to the project.
+- Using asynchronous communication channels creates a base level of passive
+  documentation that can be referenced again when similar questions come up
+  again.
+
+The goal when streamlining communication channels for InnerSource projects
+should be to align communication around topics, not around certain sets of
+people:
+
+- The project should have it's own issue tracker where structured communication,
+  decision making and progress tracking can happen transparently for all host
+  team members but also for downstream users and contributors to follow.
+- The project should have one or more discussion channels that come with less
+  rigid a structure. Typically this will be mailing lists, online fora or even
```

Comment:
```suggestion
  rigid a structure. Typically this will be mailing lists, online forums or even
```

While the existing text is grammatically correct, suggest this change.
https://grammarist.com/usage/fora-forums/